cypress/e2e/visual-regression.cy.js

```diff
@@ -0,0 +1,48 @@
+describe('Visual Regression Testing', () => {
+  let timestamp;
+
+  before(() => {
+    cy.log('Clearing screenshots before starting the test');
+    cy.task('clearScreenshots');
```

Comment:
hey hey Luna, suggestion for this slearing screenshots, The current setup clears the screenshots folder before and after the test run, meaning the baseline screenshot will be deleted if the test runs more than once. For subsequent test runs to work, you need to adjust the clearScreenshots task to not delete the baseline screenshots after they've been created.